schedule/yam/agama_iscsi_activate_unattended.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: agama_iscsi_activate_unattended
+description: >
+  Perform unattended installation with Agama iscsi activation.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/agama_auto
+  - installation/grub_test
+  - installation/first_boot
```

Comment:
we need validation of the partitioning.

Reply:
added `- console/validate_partition_table`

Reply:
yes, please add this to your ticket to not forget.